workers/reporter/src/test/kotlin/reporter/ReporterWorkerTest.kt

```diff
@@ -343,4 +346,45 @@ class ReporterWorkerTest : StringSpec({
             result shouldBe RunResult.Ignored
         }
     }
+
+    "Reporter should run without error even if there is no analyzer result" {
+        val ortRun = mockk<OrtRun> {
+            every { id } returns ORT_RUN_ID
+            every { repositoryId } returns REPOSITORY_ID
+            every { revision } returns "main"
+            every { labels } returns mapOf("projectName" to "Test project")
+        }
+
+        val ortResult = mockk<OrtResult> {
+            every { repository } returns Repository.EMPTY
+        }
+
+        val ortRunService = mockk<OrtRunService> {
+            every { getOrtRun(ORT_RUN_ID) } returns ortRun
+            every { getReporterJob(REPORTER_JOB_ID) } returns reporterJob
+            every { startReporterJob(REPORTER_JOB_ID) } returns reporterJob
+            every { generateOrtResult(ortRun, failIfRepoInfoMissing = false) } returns ortResult
+            every { storeReporterRun(any()) } just runs
+        }
+
+        val worker = ReporterWorker(
+            mockk(),
+            mockk(),
+            mockk(),
+            mockk<ReporterRunner>(),
+            ortRunService,
+            mockk()
+        )
+
+        mockkTransaction {
+            val result = worker.run(REPORTER_JOB_ID, TRACE_ID)
+
+            result shouldBe RunResult.FinishedWithIssues
+        }
+
+        val slotReporterRun = slot<ReporterRun>()
+        coVerify {
+            ortRunService.storeReporterRun(capture(slotReporterRun))
+        }
```

Comment:
Was it missed to do some checks on the reporter run captured by the slot? Otherwise, it does not make sense to use `capture` here.

Reply:
Fixed, it was a copy paste error.